<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FunctionGuardWidget</class>
 <widget class="QWidget" name="FunctionGuardWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>120</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_normal">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>120</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI Light</family>
     <pointsize>-1</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #A6B9FF;
font: 16px &quot;Microsoft YaHei UI Light&quot;;
padding-left: 18px;</string>
   </property>
   <property name="text">
    <string>账户</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QToolButton" name="assetInfoBtn">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>410</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>资产信息</string>
   </property>
  </widget>
  <widget class="QToolButton" name="keyManageBtn">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>380</y>
     <width>120</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>私钥管理</string>
   </property>
  </widget>
  <widget class="QToolButton" name="proposalBtn">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>200</y>
     <width>120</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>提案</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_normal_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>140</y>
     <width>120</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI Light</family>
     <pointsize>-1</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #A6B9FF;
font: 16px &quot;Microsoft YaHei UI Light&quot;;
padding-left: 18px;</string>
   </property>
   <property name="text">
    <string>提案</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_normal_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>320</y>
     <width>120</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI Light</family>
     <pointsize>-1</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #A6B9FF;
font: 16px &quot;Microsoft YaHei UI Light&quot;;
padding-left: 18px;</string>
   </property>
   <property name="text">
    <string>高级</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QToolButton" name="accountInfoBtn">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>120</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>账户信息</string>
   </property>
  </widget>
  <widget class="QToolButton" name="myIncomeBtn">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>90</y>
     <width>120</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>我的收益</string>
   </property>
  </widget>
  <widget class="QToolButton" name="withdrawConfirmBtn">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>260</y>
     <width>120</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>提现授权</string>
   </property>
  </widget>
  <widget class="QToolButton" name="feedPriceBtn">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>290</y>
     <width>120</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>市场喂价</string>
   </property>
  </widget>
  <widget class="QToolButton" name="coldHotTransferBtn">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>230</y>
     <width>120</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>资产变更</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
